grammar.js

```diff
@@ -444,7 +444,7 @@ module.exports = grammar({
     use: ($) =>
       seq(
         "use",
-        optional(series_of($.identifier, ",")),
+        optional(series_of($._pattern, ",")),
```

Comment:
Are multiple (or zero) comma-separated patterns allowed with `use` or was that just for identifiers?

Reply:
Multiple patterns are allowed, so your code is correct.  For posterity, the relevant line is this one: https://github.com/gleam-lang/gleam/blob/fe6dcf49a3d7787d81c8a0d32419a24b7e3ee36a/compiler-core/src/parse.rs#L723